core/src/main/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotService.scala

```diff
@@ -56,50 +57,62 @@ class SnapshotService(protected val ctx: RawlsRequestContext,
     with WorkspaceSupport
     with LazyLogging {
```

Comment:
The code added is primarily copy-paste with minor changes to use `workspaceId` instead of `namespace/name`.  Please check carefully to ensure I didn't inadvertently make additional changes.  (Or to suggest a better/cleaner way to factor out shared code paths.)